sol-anchor/convo.go

```diff
@@ -120,12 +122,26 @@ func (c *Convo) Update(msg loop.Msg) loop.Cmd {
 
 	case AskIdl:
 		return c.Action(InputIdl{}).
-			TextInput("Input the Anchor IDL in JSON format\n", "Submit").
+			TextInput(fmt.Sprintf("Paste the Anchor IDL in JSON format OR input the path of the JSON IDL in your filesystem (e.g. PATH_TO_MY_IDL/MY_IDL.json)\n"), "Submit").
 			Cmd()
 
 	case InputIdl:
+		var rawMessage string
+
+		if decoderToken, _ := json.NewDecoder(strings.NewReader(msg.Value)).Token(); decoderToken != nil {
```

Comment:
```suggestion
		// If we are able to decode first JSON token, we assume it's JSON, otherwise, we assume it's a file path
		if decoderToken, _ := json.NewDecoder(strings.NewReader(msg.Value)).Token(); decoderToken != nil {
```